integration/oasis/stationfinder/stations_iterator_alg.go

```diff
@@ -64,7 +66,7 @@ func CalcCostBetweenChargeStationsPair(from nearbyStationsIterator, to nearbySta
 		targetIDs = append(targetIDs, v.ID)
 	}
 	if len(targetPoints) == 0 {
-		err := fmt.Errorf("Empty iterator of to pass into calcCostBetweenChargeStationsPair")
+		err := fmt.Errorf("Empty iterator of to pass into CalcWeightBetweenChargeStationsPair")
```

Comment:
DON'T use capital letter at the beginning of error string. see https://github.com/golang/go/wiki/CodeReviewComments#error-strings